Dockerfile

```diff
@@ -1,33 +1,35 @@
 FROM node:6.10.0-slim
 LABEL maintainer "Code Climate <hello@codeclimate.com>"
 
-RUN apt-key adv --fetch-keys http://dl.yarnpkg.com/debian/pubkey.gpg && \
-    echo "deb http://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
-    apt-get update
+RUN adduser --uid 9000 --gecos "" --disabled-password app
 
 WORKDIR /usr/src/app
-COPY bin/docs ./bin/docs
-COPY engine.json package.json yarn.lock ./
 
-RUN mkdir /usr/local/node_modules
 ENV PREFIX=/usr/local/node_modules
 ENV PATH=$PREFIX/.bin:$PATH
 ENV NODE_PATH=$PREFIX
 ENV NPM_CONFIG_PREFIX=$PREFIX
 
-RUN apt-get install -y git jq yarn && \
+RUN mkdir $PREFIX
```

Comment:
If we don't need the layer, I'm fine with dropping it. Not sure if yarn will automatically create it for you.